k8s/crate/apptest/tester/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gcr.io/cloud-marketplace-tools/testrunner:0.1.4
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl wget netcat \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /opt/kubectl/1.21 \
+    && wget -q -O /opt/kubectl/1.21/kubectl \
```

Comment:
Sorry, i was so happy with your approvement and missed this comment. I will update kubectl version with the next nearest PR, anyway it should work because tests use very common command `kubectl exec`, nothing version-specific.